From 0f8c8f9fab210078d2fe7e70af358c39dfa2752a Mon Sep 17 00:00:00 2001
From: Haixia Shi <hshi@chromium.org>
Date: Tue, 14 Jun 2016 17:07:46 -0700
Subject: [PATCH 519/521] CHROMIUM: egl/android: Set EGL_MAX_PBUFFER_WIDTH and
 EGL_MAX_PBUFFER_HEIGHT

Set config attributes EGL_MAX_PBUFFER_WIDTH and EGL_MAX_PBUFFER_HEIGHT to
hard-coded non-zero values. These two attributes are required on Android.

BUG=b/29359794
TEST=dEQP-EGL.function.create_surface.pbuffer

Reviewed-on: https://chromium-review.googlesource.com/352700
Tested-by: Haixia Shi <hshi@chromium.org>
Reviewed-by: Tomasz Figa <tfiga@chromium.org>
(cherry picked from commit b2b27d09c1d7f694a37a80f7b04e4f79df557b1f)

BUG=b:29036398
TEST=No significant regressions in Android dEQP

Signed-off-by: Tomasz Figa <tfiga@chromium.org>

Change-Id: I56dbffcd919b71a22ec3e8d921e16212f9069934
Reviewed-on: https://chromium-review.googlesource.com/367211
Reviewed-by: Haixia Shi <hshi@chromium.org>
Tested-by: Tomasz Figa <tfiga@chromium.org>
Commit-Queue: Tomasz Figa <tfiga@chromium.org>
---
 src/egl/drivers/dri2/platform_android.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/src/egl/drivers/dri2/platform_android.c b/src/egl/drivers/dri2/platform_android.c
index 0d2efab..07b964f 100644
--- a/src/egl/drivers/dri2/platform_android.c
+++ b/src/egl/drivers/dri2/platform_android.c
@@ -773,6 +773,8 @@ droid_add_configs_for_visuals(_EGLDriver *drv, _EGLDisplay *dpy)
      EGL_NATIVE_VISUAL_TYPE, 0,
      EGL_FRAMEBUFFER_TARGET_ANDROID, 0,
      EGL_RECORDABLE_ANDROID, EGL_TRUE,
+     EGL_MAX_PBUFFER_WIDTH, 4096,
+     EGL_MAX_PBUFFER_HEIGHT, 4096,
      EGL_NONE
    };
    int count, i, j;
-- 
2.8.0.rc3.226.g39d4020

